pkg/util/hardware/hw_utils.go

```diff
@@ -56,3 +58,24 @@ func ParseCPUSetLine(cpusetLine string) (cpusList []int, err error) {
 	}
 	return
 }
+
+//GetNumberOfVCPUs returns number of vCPUs
+//It counts sockets*cores*threads
+func GetNumberOfVCPUs(cpuSpec *v1.CPU) int64 {
```

Comment:
I don't know if hardware utils is the right place for this method, as it calculates something which is related to the vmi spec, not hardware... but I won't block on it.

Reply:
I ask for that this because it was used as well in webhook, so if you see better place for that then please suggest, thanks.